static/js/src/advantage/react/components/Subscriptions/SubscriptionList/ListCard/ListCard.tsx

```diff
@@ -48,15 +48,15 @@ const ListCard = ({
       </span>
     </div>
     <Row>
-      <Col size={3}>
+      <Col size={3} small={1}>
```

Comment:
This makes the columns 1/4 in mobile, but to match the [design](https://app.zeplin.io/project/60ec5fb4e07cc90fbbf2b318/screen/60f6fa108d36bb10a7f201a0) it's supposed to be 1/3. Is there a Vanilla way of doing that or do I need to not use the grid in this case?